aten/src/ATen/native/cpu/layer_norm_kernel.cpp

```diff
@@ -29,8 +30,8 @@ void LayerNormKernelImplInternal(
   DCHECK(!gamma.defined() || gamma.numel() == N);
   DCHECK(!beta.defined() || beta.numel() == N);
   T* X_data = X.data_ptr<T>();
-  const T* gamma_data = gamma.defined() ? gamma.data_ptr<T>() : nullptr;
-  const T* beta_data = beta.defined() ? beta.data_ptr<T>() : nullptr;
+  T* gamma_data = gamma.defined() ? gamma.data_ptr<T>() : nullptr;
```

Comment:
See my above comments. It seems to me that there is no need to remove const here.

Reply:
Reverted.